### PR TITLE
Add .NET hebe implementation

### PR DESCRIPTION
**Pull request by MRmlik12**

### CODE REVIEW

File: hebe-dotnet/UonetRequestSignerHebe/Signer.cs [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Security.Cryptography;
+using System.Collections.Generic;
+
+namespace UonetRequestSignerHebe
+{
+    public class Signer
+    {
+        public SignerValues GetSignatureValues(string fingerprint, string privKey, string body, DateTime date, string path)
```
Comment (mklkj) [inline hebe-dotnet/UonetRequestSignerHebe/Signer.cs:12]: Please use the same args order as in other implementations (swap last two): https://github.com/wulkanowy/uonet-request-signer/blob/master/hebe-jvm/src/main/kotlin/signer.kt#L55-L60

File: hebe-dotnet/UonetRequestSignerHebe/Signer.cs [hunk 2]
```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Text;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Security.Cryptography;
+using System.Collections.Generic;
+
+namespace UonetRequestSignerHebe
```
Comment (mklkj) [inline hebe-dotnet/UonetRequestSignerHebe/Signer.cs:8]: Change namespace to `Wulkanowy.UonetRequestSigner.Hebe

**Review by mklkj — CHANGES_REQUESTED** [review]
Please also rebase your branch

**Comment by MRmlik12** [comment]
Ok, I renamed solution to `Wulkanowy.UonetRequestSigner.Hebe` and updated `Signer.cs` args order. I deleted `SignerValues.cs` file because I used c# 7 feature it's called "value tuples".

**Comment by MRmlik12** [comment]
I modified Signer.cs with GetEncodedPath method to encode path properly

File: hebe-dotnet/README.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Uonet+ hebe request signer for .NET
+
+## Usage
+
+```cs
+using Wulkanowy.UonetRequestSigner.Hebe;
+
+var signed = new Signer().GetSignatureValues("fingerprint", "private_key", "body", "url", DateTime.Now);
```
Comment (mklkj) [inline hebe-dotnet/README.md:8]: One last thing - maybe make the `GetSignatureValues` method `static`? Similar to `dotnet` impl

**Review by mklkj — CHANGES_REQUESTED** [review]

File: hebe-dotnet/README.md (hunk 3 above)
Comment (MRmlik12) [reply hebe-dotnet/README.md:8]: I'll try make a static class If you want

**Review by mklkj — APPROVED** [review]

**Pull request merged**